build-support/bin/generate_travis_yml.py

```diff
@@ -622,13 +621,8 @@ def make_shard(*, shard_num: int) -> Dict:
             ],
         }
         safe_append(
-            shard,
-            "env",
-            f"CACHE_NAME=integration.v1.shard_{shard_num}.py{python_version.number}{'.pantsd' if use_pantsd else ''}",
+            shard, "env", f"CACHE_NAME=integration.v1.shard_{shard_num}.py{python_version.number}",
         )
-        if use_pantsd:
-            shard["stage"] = Stage.test_cron.value
-            safe_append(shard, "env", 'USE_PANTSD_FOR_INTEGRATION_TESTS="true"')
```

Comment:
> Stop running tests with Pantsd, which fails with --chroot enabled due to too long of a prefix.

This PR been out for a while, so I'll understand if you say no, but: are you able to try this post #9636 with `--no-watchman-enable`? That should remove the prefix issue.

Reply:
I want to land this so that we can start running Py38 in the cron job. But, once #9636 lands, I can try.

Also, the Pantsd logic is wrong anyways. It was only running a subset of the tests because of the split between the V2 test runner vs. V1 test runner. It'd be better to start from scratch, I think.